packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+
+
+def invoke_check_binary(target, *args):
+    """
+    Run the dcos-checks binary.
+
+    :param target: dcos-checks subcommand
+    :param args: a list of target arguments passed to a dcos-checks binary
+    :return: combined stdout/stderr, return code
+    """
+    try:
+        output = subprocess.check_output(
+            ["/opt/mesosphere/bin/dcos-checks", "--role", "master", target] + [arg for arg in args],
```

Comment:
This role arg slightly concerns me -- it's correct right now only by coincidence, because we happen to run integration tests on a master. I don't know if we have any other tests which rely on being run on a master.

@mellenburg Do you have any guidance here? Is there a facility this test can use to e.g. SSH to a node whose role we know for certain and run this executable?